packages/medusa/src/services/event-bus.ts

```diff
@@ -1,150 +1,72 @@
-import Bull, { JobOptions } from "bull"
-import Redis from "ioredis"
-import { DeepPartial, EntityManager, In } from "typeorm"
-import { QueryDeepPartialEntity } from "typeorm/query-builder/QueryPartialEntity"
-import { ulid } from "ulid"
+import {
```

Comment:
comment: @adrien2p I would also appreciate a re-review of this file, as I have incorporated the bulk emit functionality. Same goes for the `event-bus-redis` + `event-bus-local` module services :)